src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Extensions/RoslynLspExtensions_TextEdit.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using VsLspTextEdit = Microsoft.VisualStudio.LanguageServer.Protocol.TextEdit;
+
+namespace Roslyn.LanguageServer.Protocol;
+
+internal partial class RoslynLspExtensions
+{
+    public static VsLspTextEdit ToVsLspTextEdit(this TextEdit textEdit)
```

Comment:
Please don't create this extension method. Keeping `RoslynLspExtensions` and `VsLspExtensions` fully completetly distinct is intentional so that it its easier to remove `VsLspExtensions` later.

Instead of adding this, at the call sites where you used this helper, just call `VsLspFactory.CreateTextEdit(...)`.

Reply:
I believe all of those are addressed, please re-activate if I missed something.